### ISSUE TITLE
BoundsError on getting help

### ISSUE THREAD

**Issue by ssfrr** [timholy/Revise.jl#404]
Not sure if I can reproduce this, I think it was after a combination of
1. `using Revise`
2. `using ...` (a bunch of other packages)
3. evaluating some code
4. doing a `pkg> update`
5. `pkg> add Combinatorics`
6. `help?> combinations`

```
help?> combinations
ERROR: BoundsError: attempt to access 114-element Array{Array{Int64,1},1} at index [0]search:
Stacktrace: 
c [1] ogetindexm at b./array.jl:744i [inlined]n
a [2] tpush!i at o/home/sfr/.julia/packages/Revise/0KQ7U/src/backedges.jl:54n [inlined]s
  [3] madd_to_backedges!u(l::tRevisei.sBackEdgese, t::_Arrayc{oRevisem.bSlotDepi,n1a}t, i::oInt64n, s:: JuliaInterpreterC.oSlotNumbero)l at L/home/sfr/.julia/packages/Revise/2NCCI/src/backedges.jl:80e
x [4] CRevise.BackEdgeso(m::bCorei.nCodeInfoa)t at i/home/sfr/.julia/packages/Revise/2NCCI/src/backedges.jl:181o
n [5] sminimal_evaluation! (w::iReviset.hCodeTrackingMethodInfo_, r::eJuliaInterpreterp.lFramea)c at e/home/sfr/.julia/packages/Revise/2NCCI/src/lowered.jl:33m
e [6] n#methods_by_execution!#15t(_::cBoolo, m::bBooli, n::aBase.Iteratorst.iPairso{nUnion{}s, Union{}C,oTuplem{b}i,nNamedTuplea{t(o)r,iTuplec{s}
}
}, ::typeof(Revise.methods_by_execution!), ::Any, ::Revise.CodeTrackingMethodInfo, ::Dict{Module,Array{Expr,1}}, ::Module, ::Expr) at /home/sfr/.julia/packages/Revise/0KQ7U/src/lowered.jl:74
 [7] #methods_by_execution! at ./none:0 [inlined]
 [8] #eval_with_signatures#70 at /home/sfr/.julia/packages/Revise/0KQ7U/src/Revise.jl:392 [inlined]
 [9] #eval_with_signatures at ./none:0 [inlined]
 [10] #instantiate_sigs!#71(::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Revise.instantiate_sigs!), ::OrderedCollections.OrderedDict{Module,OrderedCollections.OrderedDict{Revise.RelocatableExpr,Union{Nothing, Array{Any,1}}}}) at /home/sfr/.julia/packages/Revise/2NCCI/src/Revise.jl:403
 [11] instantiate_sigs! at /home/sfr/.julia/packages/Revise/0KQ7U/src/Revise.jl:397 [inlined]
 [12] maybe_parse_from_cache!(::Revise.PkgData, ::String) at /home/sfr/.julia/packages/Revise/2NCCI/src/pkgs.jl:266
 [13] (::Revise.var"#38#39"{String})() at /home/sfr/.julia/packages/Revise/0KQ7U/src/pkgs.jl:449
 [14] with_logstate(::Revise.var"#38#39"{String}, ::Base.CoreLogging.LogState) at ./logging.jl:395
 [15] with_logger at ./logging.jl:491 [inlined]
 [16] watch_manifest(::String) at /home/sfr/.julia/packages/Revise/2NCCI/src/pkgs.jl:428
 [17] (::Revise.Rescheduler{typeof(Revise.watch_manifest),Tuple{String}})() at /home/sfr/.julia/packages/Revise/2NCCI/src/types.jl:245
```

edit: the `update` upgrade Revise from v2.3.1 to v2.3.2. It seems pretty reasonable that things might be a bit finicky after updating Revise in a session (can Revise revise Revise?) but I figured I'd report anyways.

**Comment by timholy** [comment]
That bug is specifically fixed in 2.3.2 (see #401), so hopefully it was just something that happened during the upgrade.

**Comment by timholy** [comment]
But thanks for reporting!
